build.gradle.kts

```diff
@@ -66,7 +66,7 @@ plugins {
     kotlin("jvm") version "1.8.21"
     application
 
-    id("org.jetbrains.intellij") version "1.14.0"
+    id("org.jetbrains.intellij") version "1.14.2"
```

Comment:
Had to upgrade in order to make the project compile on my pc due to a strange problem with local paths resolution...